### ISSUE TITLE
League Community Incident ??

### ISSUE THREAD

**Issue by in567k** [R3nzTheCodeGOD/R3nzSkin#1191]
The fact that people got the official skins in fantom files and everyone is claiming they're "making an app" while all they're doing is growing their discord servers and giving only a limited portion of people access to content is probably one of league community's incidents lol 
you guys do realize these are just skins that only you can see and there's zero need to act like it's a big deal to just post them without all the bluffing and stalling right ? 
league community got no cure and that's just simple facts guess i'll just add what yall doin rn to the list lmao

**Comment by JulienLOOA** [comment]
U send fantome files ?

**Comment by JinkiraY** [comment]
![image](https://github.com/R3nzTheCodeGOD/R3nzSkin/assets/171052446/61fceffa-6c72-4bb3-80aa-9feb15466b92)
well i give fantome files for free just dm me
dc : powowzx

**Comment by lol123love** [comment]
> The fact that people got the official skins in fantom files and everyone is claiming they're "making an app" while all they're doing is growing their discord servers and giving only a limited portion of people access to content is probably one of league community's incidents lol you guys do realize these are just skins that only you can see and there's zero need to act like it's a big deal to just post them without all the bluffing and stalling right ? league community got no cure and that's just simple facts guess i'll just add what yall doin rn to the list lmao

Because extracting every skin took a lot of time from our lives and a lot of coding, You might find it simple hah it's just a file! no it's not, if so make it urself and stop crying 😀

**Comment by FishB0nes98** [comment]
> > The fact that people got the official skins in fantom files and everyone is claiming they're "making an app" while all they're doing is growing their discord servers and giving only a limited portion of people access to content is probably one of league community's incidents lol you guys do realize these are just skins that only you can see and there's zero need to act like it's a big deal to just post them without all the bluffing and stalling right ? league community got no cure and that's just simple facts guess i'll just add what yall doin rn to the list lmao
> 
> Because extracting every skin took a lot of time from our lives and a lot of coding, You might find it simple hah it's just a file! no it's not, if so make it urself and stop crying 😀

I have to agree with you. I tried to port a syndra skin yesterday, and however it was successful it took me 5 hours because of my pretty bad Python knowledge + I had to learn all the programs.

**Comment by T9Jungler** [comment]
ainz relesing his app in 10 days so its not scam, i tested closed beta and it works so you are not right
